react/next/components/SliderTrack.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react'
+import { transitionType } from '../typings/global'
+
+interface Props {
+  className?: string
+  transform: number
+  transition: transitionType
+}
+
+const SliderTrack: FC<Props> = ({
+  children,
+  className,
+  transform,
+  transition,
+}) => (
+  <div
+    className={`${className} flex relative pa0 ma0`}
+    style={{
+      willChange: 'transform',
+      transition: `transform ${transition.speed}ms ${transition.timing}`,
+      transitionDelay: `${transition.delay}ms`,
+      transform: `translate3d(${transform}px, 0, 0)`,
+    }}
+    id="slider-items"
```

Comment:
this won't work nicely if we have 2 sliders on the page, right? e.g. a shelf and a banner carousel

Reply:
I'm generating this id with the passed label now. Do you think that is a good solution?

Reply:
sure, we just need to highlight in some way that the label is required and need to be unique